### DISCUSSION TITLE
How to disable /health endpoint for tracing

### DISCUSSION THREAD

**Discussion by knowpritam** [micronaut-projects/micronaut-core#5203] [Q&A]
We are using jaegar for tracing and I can see that by default the /health endpoint is traced and hence the jaegar UI becomes noisy as this endpoint is hit every 1 min (as per micronaut defalts) and hence every 1 min we see a new entry on jeager UI.

Any idea how do we disable/skip tracing for a particular endpoint or /health endpoint to be precise ?

application.yml entry for tracing
"
tracing:
  jaeger:
    enabled: ${TRACING_JAEGER_ENABLED:true}
    enable-b3-propogation: true
    sampler:
      probability: ${TRACING_JAEGER_SAMPLER_PROBABILITY:1}
    sender:
      agentHost: ${TRACING_JAEGER_SENDER_AGENT_HOST:localhost}
      agentPort: ${TRACING_JAEGER_SENDER_AGENT_PORT:5775}
    reporter:
      flushInterval: 500 # interval(ms) to flush all in-memory spans to the Agent/Collector (default: 1000)
      maxQueueSize: 200 #max size of buffer to store spans in memory before sending them to Agent/Collector(default: 100)
"

![image](https://user-images.githubusercontent.com/64675361/113246203-27018600-92d6-11eb-9eaa-196db5ff1179.png)

I can see one similar issue but there is no traction on the same from last 10 months
https://github.com/micronaut-projects/micronaut-core/issues/1296

**Comment by jameskleeh** [comment]
@knowpritam It seems to me #1296 would cover what you're talking about. I don't believe the functionality exists today to be able to exclude certain endpoints from tracing. This is functionality we are open to including but it has not been prioritized as of today. I don't know when that may happen but you are welcomed to submit a PR to include it. If you have any questions or wish to continue the discussion further it should happen in the issue so the discussion is in one place.
